README.md

```diff
@@ -116,6 +116,23 @@ The `wifi` section below includes the following entries:
     the data folder
   * `username`, `password` - EAP login credentials
 
+The `eth` section below includes the following entries:
+```json
+{
+  "eth": {
+    "enabled": true,
```

Comment:
I think the `enabled` flag is redundant. From what I gather from the commit we support either WiFi or Etherenet. So the documentation should state that the configuration should include either the `wifi` field or `eth`. Whichever exists is used.